scripts/ci-conformance.sh

```diff
@@ -93,4 +94,10 @@ cleanup() {
 
 trap cleanup EXIT
 
-make test-conformance
+if [[ "${WINDOWS}" == "true" ]]; then
+  # Windows isn't really conformance, we typically run at 4 to keep the time reasonable (~45 mins)
+  export CONFORMANCE_NODES=4
```

Comment:
Let's define runtime variables like `CONFORMANCE_NODES` in https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml#L218-L252. In case we want to run [Serial] tests in the future, we can simply define an environment variable in the job configuration

Reply:
good call, I'll update that there

Reply:
https://github.com/kubernetes/test-infra/pull/22180